sympy/core/compatibility.py

```diff
@@ -676,6 +677,36 @@ def ordered(seq, keys=None, default=True, warn=False):
         yield from d[k]
         d.pop(k)
 
+class OpDispatcher(Dispatcher):
+    """
+    Dispatcher for non-associative operations, e.g. pow
+
+    .. notes::
+       This approach is experimental, and can be replaced or deleted in the future.
+       See https://github.com/sympy/sympy/pull/19463.
+
+    See Also
+    ========
+
+    core.operations.AssocOpDispatcher
+
+    """
+    def __init__(self, name, fallback, doc=None):
+        self.fallback = fallback
+        self.handlermethod = "_%s_handler" % name
```

Comment:
Is this used?

Reply:
This makes `power(2,3)` return `Pow(2,3)` (which is 8) without registering `(int, int)` to `power`.

Reply:
I'm talking about `handlermethod`. This class never uses it.

Reply:
Oh, you're right. This class doesn't need it anymore.